frontend/javascripts/oxalis/model/reducers/volumetracing_reducer.ts

```diff
@@ -377,6 +377,12 @@ function VolumeTracingReducer(
       return expandSegmentParents(state, action);
     }
 
+    case "SET_VOLUME_BUCKET_DATA_HAS_CHANGED": {
+      return updateVolumeTracing(state, action.layerName, {
+        volumeBucketDataHasChanged: true,
+      });
```

Comment:
`updateVolumeTracing` requires a `volumeTracingId` as a second argument

```suggestion
      return updateVolumeTracing(state, action.volumeTracingId, {
        volumeBucketDataHasChanged: true,
      });
```

Reply:
I gave this a debugging double check: The `volumeBucketDataHasChanged` field is never updated by the frontend. The warning only appears due to other reasons. See my comment in the LoadMeshMenuItemLabel